app/src/main/java/pub/devrel/easypermissions/sample/RationaleFragment.java

```diff
@@ -0,0 +1,93 @@
+package pub.devrel.easypermissions.sample;
+
+import android.Manifest;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import pub.devrel.easypermissions.AfterPermissionGranted;
+import pub.devrel.easypermissions.EasyPermissions;
+
+import java.util.List;
+
+public class RationaleFragment extends Fragment implements EasyPermissions.PermissionCallbacks,
```

Comment:
Is there any need to introduce a whole new Fragment here rather than just adding this to the existing sample?

Would make the diff much smaller to just add the new callbacks to our existing sample code.

Reply:
So, I added `RationaleFragment ` in connection to my initial implementation of RationaleCallbacks which extends PermissionCallbacks. I guess it's not needed anymore. Thanks for your feedback @SUPERCILEX . Will update this PR shortly.